# 1. Workaround: Host python can not find python-dasbus
# 2. Prevent running pygobject scripts from termux prefix

--- a/meson.build
+++ b/meson.build
@@ -20,7 +20,7 @@
 
 # Hard Python module dependencies
 dasbus_result = python.find_installation('python3', modules:['dasbus'], required: false)
-if not dasbus_result.found()
+if false and not dasbus_result.found()
   error('dasbus is required for D-Bus remote controller interface')
 endif
 
@@ -37,7 +37,7 @@
 gtk_test = run_command(python3, '-c', gtk_command, check: false)
 description = f'GTK @gtk_major_version@.@gtk_minor_version@'
 version = gtk_test.stdout().strip()
-if gtk_test.returncode() != 0
+if meson.can_run_host_binaries() and gtk_test.returncode() != 0
   error(f'@description@ failed (found @version@)')
 endif
 
